base/server/src/main/java/org/dogtagpki/server/rest/v2/PKIServlet.java

```diff
@@ -189,7 +158,7 @@ public Method getActionMethod(HttpMethod met, String path) {
         }
         String keyPath = webActions.keySet().stream().
                 filter( key -> {
-                    String keyRegex = key.replace("{}", "([A-Za-z0-9_\\-]+)");
+                    String keyRegex = key.replace("{}", "([A-Za-z0-9_\\-\\.\\\s]+)");
```

Comment:
Yes, I was looking at old URL specification not considering non ASCII. I have update to get everything until `/` following your comment since there is no need to verify which characters are in the path, the URL should already be validated by tomcat.